### DISCUSSION TITLE
Lightning talk at ESIP

### DISCUSSION THREAD

**Discussion by kthyng** [xarray-contrib/cf-xarray#356] [General]
Hi developers esp @dcherian @malmans2,

I've been asked to give a lightning talk at [ESIP](https://www.esipfed.org/) Aug 11th. I haven't represented `cf-xarray` before (other than telling everyone about it!) so let me know if you have any concerns, and alternatively, let me know if you have any materials to present from!

**Comment by dcherian** [comment]
Nice! glad to hear you're doing it.

Here are slides I presented at AMS in 2021: https://docs.google.com/presentation/d/1N5Ys80YVHndcfqxbwrFKj3Oa6owMa5uFxaTDJmC7zn0/edit#slide=id.p

The other material is under "In depth examples" in the docs: https://cf-xarray.readthedocs.io/en/latest/

It'd be nice to illustrate how you use it at work if there's time to prepare that.

**Reply by kthyng** [reply]
Ah, you must be part of ESIP. I've been meaning to get involved as I hear about the group periodically, but haven't yet. Thanks, I'll look at this material as I prepare, and yes I'll include a demo of how I use it.

**Comment by malmans2** [comment]
Hi! I also got the invitation to present `cf-xarray` at ESIP, but I won't be able to present that day.
Good news that @kthyng can present though, let me know if you need anything from me!

**Reply by kthyng** [reply]
Sounds good! I'll try to start the talk tomorrow so I have time to check in with you two beforehand if needed.
